bin/install

```diff
@@ -51,4 +138,4 @@ install() {
 }
 
 #
-install "$ASDF_INSTALL_TYPE" "$ASDF_INSTALL_VERSION" "$ASDF_INSTALL_PATH"
\ No newline at end of file
+install "$ASDF_INSTALL_TYPE" "$ASDF_INSTALL_VERSION" "$ASDF_INSTALL_PATH"
```

Comment:
Accidental newline. What is the standard in this project on newlines at the end of files?